source/mapping-rules/mapping-rule-options/time-series.txt

```diff
@@ -0,0 +1,181 @@
+  collections improves query efficiency and reduces the disk usage for
+  time series data and :term:`secondary indexes <secondary index>`.
+
+- Use cases for time series collections include 
+  :abbr:`IoT (Internet of Things)`, forecasting, and time based analysis.
+
+Before you Begin
+----------------
+
+- For full details on each time series field component, see 
+  :ref:`time-series-fields`.
+
+- If the time series data in your sync job does not require any 
+  complex transformations you may benefit from migrating your time 
+  series data as a separate sync job. This can make it easier to 
+  preload a large time series collection and once completed migrate 
+  the rest of your data.
+
+- If your migrating larges amounts of data, consider using 
+  :ref:`table filters <rm-table-filters>` to batch your sync jobs 
+  based on logical conditions. For example, you might import one year 
+  of historical data at a time using table filters. This allows a large 
+  job to be broken into more manageable segments.
+
+- If a long-running sync job fails unexpectedly (for example, due to a 
+  database outage) and is not able to automatically recover, you may 
+  be able to use :ref:`table filters <rm-table-filters>` to create a 
+  new sync job that picks up from where the previous one left off. This 
+  approach saves you from re-importing your previous data from scratch.
+
+Steps
+-----
+
+.. procedure::
+   :style:  connected
+
+   .. step:: Select a collection with a datetime field 
+
+      a. Navigate to the :guilabel:`Mapping` screen.
+      #. On the :guilabel:`Schema model` pane under the 
+         :guilabel:`MongoDB` header, click on a collection that has a 
+         datetime field.
+
+   .. step:: Configure the time series collection
+
+      a. On the :guilabel:`Mappings` pane, click the :icon-lg:`Edit` 
+         :guilabel:`Configure` button.
+      #. Select the :guilabel:`Time Series Collection` radio button.
+      #. Select a :guilabel:`Time Field`.
+
+         ``Time Field`` is the name of the field which contains the 
+         datetime field in each time series document. Documents in a 
+         time series collection must have a valid BSON date data type. 
+         For details on converting a field to a datetime type in 
+         Relational Migrator, see 
+         :ref:`field customizations <rm-field-customizations>`.
+
+         .. note::
+
+            If you have multiple datetime columns must select a 
+            single field per timeseries collection.
+
+      #. (Optional) Select a :guilabel:`Meta Field`.
+
+         ``Meta Field`` is the name of the field which contains metadata
+         in each time series document. The metadata in the specified 
+         field should be data that is used to label a unique series of
+         documents. The metadata should rarely, if ever, change.
+
+      #. Select a :guilabel:`Granularity`.
+
+         ``Granularity`` should be a value that most closely matches 
```

Comment:
Suggest incorporating Vale's feedback to avoid "should" here. Consider:

"Specify a ``Granularity`` value that most closely matches..."